_search-plugins/search-relevance/index.md

```diff
@@ -98,4 +98,60 @@ Setting `size` to a high value (for example, larger than 250 documents) may degr
 {: .note}
 
 You cannot save a given comparison for future use, so Compare Search Results is not suitable for systematic testing.
-{: .note}
\ No newline at end of file
+{: .note}
+
+## Comparing OpenSearch search results with re-ranked results
+
+One use case of Compare Search Results is to compare raw OpenSearch results with the same results processed by a re-ranking application. An example of such a re-ranker is **Kendra Intelligent Ranking for OpenSearch**, contributed by the Amazon Kendra team. This plugin takes search results from OpenSearch and applies Kendra’s semantic relevance rankings calculated using vector embeddings and other semantic search techniques. For many applications, this gives better result rankings.
+
+To try Kendra Intelligent Ranking, you must first set up the Amazon Kendra service. To get started, see [Amazon Kendra](https://aws.amazon.com/kendra/). For detailed information, including plugin setup instructions, see [Intelligently ranking OpenSearch (self managed) results using Amazon Kendra](https://docs.aws.amazon.com/kendra/latest/dg/opensearch-rerank.html).
+
+Once you've set up Kendra Intelligent Ranking, enter a query in **Query 1** and the same query using the Kendra Intelligent Ranking service for re-ranking in **Query 2**. Then compare search results from OpenSearch and Amazon Kendra.
```

Comment:
```suggestion
Once you've set up Kendra Intelligent Ranking, enter a query in **Query 1** and enter the same query using the Kendra Intelligent Ranking service for re-ranking in **Query 2**. Then compare the search results from OpenSearch and Amazon Kendra.
```

Reply:
Can this read "...and enter the same query using Kendra Intelligent Ranking in **Query 2**."?